README.md

```diff
@@ -223,24 +239,28 @@ Let's change the image from *redis:3.0.5* to *redis:3.0.6* (e.g. upgrade):
 
 ```
 $ swarmctl service update redis --image redis:3.0.6
-89831rq7oplzp6oqcqoswquf2
+dy0vwwae8ropb5skvzcekzk6o
 
 $ swarmctl service inspect redis
-ID                : 89831rq7oplzp6oqcqoswquf2
-Name              : redis
-Replicas          : 6
+ID                   : dy0vwwae8ropb5skvzcekzk6o
+Name                 : redis
+Replicas             : 0/6
+Update Status
+ State               : UPDATING
+ Started             : 28 seconds ago
+ Message             : update in progress
 Template
  Container
-  Image           : redis:3.0.6
-
-Task ID                      Service    Instance    Image          Desired State    Last State                Node
--------                      -------    --------    -----          -------------    ----------                ----
-7947mlunwz2dmlet3c7h84ln3    redis      1           redis:3.0.6    RUNNING          RUNNING 34 seconds ago    node-3
-56rcujrassh7tlljp3k76etyw    redis      2           redis:3.0.6    RUNNING          RUNNING 34 seconds ago    node-1
-8l7bwrduq80pkq9tu4bsd95p4    redis      3           redis:3.0.6    RUNNING          RUNNING 36 seconds ago    node-2
-3xb1jxytdo07mqccadt06rgi0    redis      4           redis:3.0.6    RUNNING          RUNNING 34 seconds ago    node-1
-16aate5akcimsye9cp5xis1ih    redis      5           redis:3.0.6    RUNNING          RUNNING 34 seconds ago    node-2
-dws408a3gz0zx0bygq3aj0ztk    redis      6           redis:3.0.6    RUNNING          RUNNING 34 seconds ago    node-3
+  Image              : redis:3.0.6
+
+Task ID                      Service    Slot    Image          Desired State    Last State                  Node
+-------                      -------    ----    -----          -------------    ----------                  ----
+1v7x9wc383ut37dw4cwqkz9v4    redis      1       redis:3.0.6    RUNNING          PREPARING 25 seconds ago    node-1
+a1rw3a7vhz3y1jbhfnzd55oft    redis      2       redis:3.0.6    RUNNING          PREPARING 25 seconds ago    node-3
+6bl8maqveqep7cbvfz5dmix8o    redis      3       redis:3.0.6    RUNNING          PREPARING 25 seconds ago    node-1
+3jv0r7yctyz4edryz1gq5sw0c    redis      4       redis:3.0.6    RUNNING          PREPARING 25 seconds ago    node-2
+139jz9jqvwu6i3z6akn3geebs    redis      5       redis:3.0.6    RUNNING          PREPARING 25 seconds ago    node-2
+34bn0omf5cjkrn9flunh0eei7    redis      6       redis:3.0.6    RUNNING          PREPARING 25 seconds ago    node-3
```

Comment:
This shows them all in the `PREPARING` state. Maybe the image is still being pulled?

I would expect the example to show them `RUNNING` after the upgrade.